stable/faucet/faucet/README.md

```diff
@@ -1,6 +1,6 @@
 # Faucet SDN Controller
 
-This chart contains a basic installation of Faucet packaged for SLATE.
+This chart contains a basic installation of Faucet packaged for SLATE. Faucet takes in a given yaml configuration document (more details below) and uses its info to tell a set of OpenFlow switches what the topology and rules of their network are. This gives the user a granular level of control of their network that they can change at will, assuming the network is built with faucet complient switches.
```

Comment:
Faucet takes in a given yaml configuration document (more details below) and uses its info to tell a set of OpenFlow switches what the topology and rules of their network are.

Can we find a better way to phrase this?